lib/elliptic/eddsa/key.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var elliptic = require('../../elliptic');
+var utils = elliptic.utils;
+var assert = utils.assert;
+
+function KeyPair(eddsa, params) {
+  this.eddsa = eddsa;
+  this.pubBytes = params.pubBytes;
```

Comment:
It can certainly be done, but then you'd need to get at everything through
accessors. Not necessarily a bad thing of course ...

Note that the public key bytes are hashed when signing, implying a Point
along the way, so I don't think there's much to be gained, by doing it
lazily.

But I can make it use accessors and do lazy member computation if you want.